simple_https_client/src/lib.rs

```diff
@@ -16,7 +16,7 @@ use record::{AppError, RecordStream};
 use tracing::info;
 
 #[allow(dead_code)]
-pub const SHA256_Aes128Gcm_EcdsaSecp256r1Sha256_X25519: Algorithms = Algorithms(
+pub const SHA256_Aes128Gcm_EcdsaSecp256r1Sha256_X25519: Algorithms = Algorithms::new(
```

Comment:
I see this modification in several places. Is this because it is more hygienic for Rust or is it needed for hax.

Reply:
Making structs visible everywhere is generally not great. Especially when using named structs, not using constructors adds noise when changing the struct. So yeah, general Rust/engineering hygiene.